BUILD.sh

```diff
@@ -96,12 +96,14 @@ fi
 
 # Some clusters have compute nodes with slightly different hostnames, so we
 # set MY_CLUSTER appropriately
-if [[ $MY_CLUSTER =~ newell* ]]; then
+if [[ $MY_CLUSTER =~ ^newell.* ]]; then
   export MY_CLUSTER=newell
-elif [[ $MY_CLUSTER =~ dl* ]]; then
+elif [[ $MY_CLUSTER =~ ^dl.* ]]; then
   export MY_CLUSTER=marianas
-elif [[ $MY_CLUSTER =~ dmi* ]]; then
+elif [[ $MY_CLUSTER =~ ^dmi.* ]]; then
   export MY_CLUSTER=incline
+elif [[ $MY_CLUSTER =~ ^dane.* ]]; then
+  export MY_CLUSTER=dane
```

Comment:
@cameronrutherford I believe this is what you want to do. The previous regular expression is problematic, as `dane` matches `dl*` and then modules are loaded from  `marianasVariable.sh`